qemu/tests/numa_consistency.py

```diff
@@ -1,125 +1,105 @@
-import resource
+import os
 
 from avocado.utils import process
 
-from virttest import cpu
+from virttest import data_dir
 from virttest import env_process
-from virttest import error_context
 from virttest import utils_misc
 from virttest import utils_test
-from virttest import arch
 from virttest.staging import utils_memory
 
 
-@error_context.context_aware
```

Comment:
We need it.

Reply:
@PaulYuuu so for me to be aware, why is better using error_context instead of test.log directly?

Reply:
test.log only log a single line, but error_context can log the context.

```
Context: Reboot guest 'avocado-vt-vm1'.
Context: Reboot guest 'avocado-vt-vm1'. --> rebooting 'avocado-vt-vm1'
Context: Reboot guest 'avocado-vt-vm1'. --> rebooting 'avocado-vt-vm1' --> waiting for guest to go down
Context: Reboot guest 'avocado-vt-vm1'. --> rebooting 'avocado-vt-vm1' --> logging in after reboot
```

```
INFO | Running 'qemu-img info /usr/share/edk2/aarch64/QEMU_EFI-silent-pflash.qcow2 -U'
```

So in short, you can use test.log to print some info for debugging or note, but use error_context to tell what's the test case doing now step by step. error_context.base_context can reset the thread.

Reply:
Even the test case does not use error_context.context, the framework will also use it.

Reply:
Got your point, it is a good way to follow the all the steps through the case, thanks @PaulYuuu

Reply:
Updated to use error_context !

Reply:
```
[stdlog] 2023-11-02 07:50:08,923 avocado.test INFO | Context: Executing stress test in node: 0
[stdlog] 2023-11-02 07:50:08,923 avocado.test INFO | Context: Executing stress test in node: 0 --> Get the qemu memory use for node: 1
```

Reply:
@PaulYuuu sorry for the late commit, @PaulYuuu could you review again the PR? Thanks !